tests/test_transforms/test_encoders/test_mean_encoder_transform.py

```diff
@@ -31,7 +31,7 @@ def category_ts() -> TSDataset:
 def expected_micro_category_ts() -> TSDataset:
     df = generate_ar_df(start_time="2001-01-01", periods=6, n_segments=2)
     df.rename(columns={"target": "mean_encoded_regressor"}, inplace=True)
-    df["mean_encoded_regressor"] = [np.NaN, 1, 1.5, 1.5, 2.75, 2.25] + [np.NaN, 6.0, 6.25, 7, 7.625, 8.0]
+    df["mean_encoded_regressor"] = [np.NaN, np.NaN, np.NaN, 1.5, 2.75, 2.25] + [np.NaN, np.NaN, 6.25, 7, 7.625, np.NaN]
```

Comment:
Why the second value is `np.NaN`? As I can see, it is a first value of category "B". 

As I understand, parameter `handle_missing` is only about missing values in `in_column`, not about first value of category or missing values in `target`.

Reply:
There were not timestamps with `in_column="B"` before second timestamp, so value is NaN